arangopipe/arangopipe/arangopipe_storage/arangopipe_api.py

```diff
@@ -216,6 +240,18 @@ def register_dataset(self, ds_info, user_id="authorized_user"):
     def register_featureset(self, fs_info, dataset_id, \
                             user_id = "authorized_user"):
         """ Register a featureset. ManagedServiceConnParamThe operation requires specifying a user id. If the user id is permitted to register a featureset, then the registration proceeds, otherwise an unauthorized operation is indicated. """
+        fs_name = fs_info["name"]
```

Comment:
Can we move that into general function checking a collection?
I am just thinking about adding new entity types

Reply:
@joerg84 . So I will refactor to a single template for all entity types.